fab/environments.yml

```diff
@@ -273,13 +278,18 @@ production:
         concurrency: 1
     hqcelery1:
       reminder_case_update_queue:
-        concurrency: 4
+        pooling: gevent
+        concurrency: 5
+        num_workers: 2
       reminder_queue:
-        concurrency: 8
+        pooling: gevent
+        concurrency: 5
+        num_workers: 2
       reminder_rule_queue:
         concurrency: 1
         max_tasks_per_child: 1
       sms_queue:
+        pooling: gevent
         concurrency: 8
```

Comment:
leaving this at 8 because we have one sms gateway that doesn't allow more than 8 concurrent connections, so i need to make a code update before we increase this on prod